### DISCUSSION TITLE
Why do I get skybox posterization?

### DISCUSSION THREAD

**Discussion by Barugon** [FyroxEngine/Fyrox#263] [Q&A] [answered]
In the FPS tutorial, here's what my sky looks like.
![Screenshot from 2021-12-19 22-16-08](https://user-images.githubusercontent.com/16503728/146720710-db1bf89b-7ead-4fa5-a338-05ee30689166.png)

The textures don't have that much posterization.
![front](https://user-images.githubusercontent.com/16503728/146720772-f8548c02-3304-46a2-acb9-b7c3cedf9e6c.jpg)

Am I doing something wrong?

**Comment by mrDIMAS** [comment, marked as answer]
Looks like texture compression artifacts, you can disable this by calling 

```rust
engine.resource_manager.state().set_textures_import_options(
    TextureImportOptions::default().with_compression(CompressionOptions::NoCompression),
);
```

right after engine initialization.

**Reply by Barugon** [reply]
Thanks. I'll give this a try.

**Reply by Barugon** [reply]
That cleaned it up.
![Screenshot from 2021-12-21 01-49-46](https://user-images.githubusercontent.com/16503728/146909141-e79fb2bb-f135-49a6-a807-c1a483138f72.png)

**Reply by Barugon** [reply]
Is it possible to import some textures with compression and some without?

**Reply by Barugon** [reply]
Never mind, figured it out.
